android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -466,14 +463,20 @@ class XMTPModule : Module() {
                 .map { DecodedMessageWrapper.encode(it) }
         }
 
-        AsyncFunction("groupMessages") { clientAddress: String, id: String ->
+        AsyncFunction("groupMessages") { clientAddress: String, id: String, limit: Int?, before: Long?, after: Long?, direction: String? ->
             logV("groupMessages")
             val client = clients[clientAddress] ?: throw XMTPException("No client")
-            if (client.libXMTPClient == null) {
-                throw XMTPException("Create client with enableAlphaMLS true in order to synch groups")
-            }
```

Comment:
I made `libXMTPClient` private in the android sdk like iOS so that integrators can't call the rust library directly without going through the sdk protections. This means we can no longer do this check in RN. But that should be oaky because it will throw from Android if they don't have it setup.